### ISSUE TITLE
Consider 2D kernels?

### ISSUE THREAD

**Issue by jcohenadad** [ivadomed/ms-multi-spine-challenge-2024#22]
Given that the strategy seems to lean towards training on the raw, thick slices, dataset (see #21), and that most ground truth labels seem to only span a single slice (with very few exceptions), maybe training a 2D vs. a 3D model would be more performant?

**Comment by plbenveniste** [comment]
That's definitely something to investigate! I would say this is overall a great method for "detecting" lesions. However, in my experience, it sometimes misses some lesion voxels on R-L extremities of the lesions.

**Comment by jcohenadad** [comment]
> it sometimes misses some lesion voxels on R-L extremities of the lesions

right, but looking at some of the ground truth labels for this challenge, these extreme R-L voxels are also "missed" ;)

**Comment by plbenveniste** [comment]
Ultimately, I did not investigate this since I focused on fine-tuning existing models (my ACTRIMS model and the MultiTalent model) which are all 3d. @tomDag25 did you investigate this yourself?

**Comment by tomDag25** [comment]
I didn't have time yet but I'm launching one today. I will use dataset 17 (multimodal preproc in T2raw space) to test it.

**Comment by plbenveniste** [comment]
This was explored in both mono-modal and multi-modal case. It only proved effective in the multi-modal case.
